[topological_utils] CMake Error `LoadPointSet.py` missing

from https://lcas.lincoln.ac.uk/jenkins/job/ros-indigo-topological-utils_binarydeb_trusty_amd64/6/console

```
CMake Error at cmake_install.cmake:87 (FILE):
  file INSTALL cannot find
  "/tmp/buildd/ros-indigo-topological-utils-0.0.27-0trusty-20150320-1108/scripts/LoadPointSet.py".
```

Obviously, that file doesn't exist but it's trying to install it. Please, before stating that something is ready for release, everybody must run `catkin_make install` locally. It's such a waste of time to have these things fail only after things have been released.

--- a/topological_utils/CMakeLists.txt
+++ b/topological_utils/CMakeLists.txt
@@ -78,7 +78,6 @@ catkin_package(
    scripts/crop_map.py
    scripts/insert_map.py
    scripts/list_maps
-   scripts/LoadPointSet.py
    scripts/map_collection_change.py
    scripts/map_export.py
    scripts/node_rm.py

